### PR TITLE
tiny update foreign key doc

### PR DESCRIPTION
**Pull request by crazycs520**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v7.0 (TiDB 7.0 versions)
- [x] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by easonn7** [comment]
LGTM

**Review by Oreoxmt — APPROVED** [review]
LGTM

File: foreign-key.md [hunk 1]
```diff
@@ -298,7 +298,7 @@ Create Table | CREATE TABLE `child` (
 ### 与 TiDB 工具的兼容性
 
 - [TiDB Binlog](/tidb-binlog/tidb-binlog-overview.md) 不支持外键功能。
-- [DM](/dm/dm-overview.md) v6.6.0 在同步数据到下游 TiDB 时，会显式关闭下游 TiDB 的 [`foreign_key_checks`](/system-variables.md#foreign_key_checks)，所以由外键产生的级联操作不会从上游同步到下游，进而导致上下游数据不一致。这与 v6.6.0 之前版本 DM 的行为一致，因为在 v6.6.0 之前 TiDB 不支持外键。
+- [DM](/dm/dm-overview.md) 不兼容外键功能。DM v6.6.0 在同步数据到下游 TiDB 时，会显式关闭下游 TiDB 的 [`foreign_key_checks`](/system-variables.md#foreign_key_checks)，所以由外键产生的级联操作不会从上游同步到下游，进而导致上下游数据不一致。这与 v6.6.0 之前版本 DM 的行为一致。
```
Comment (Oreoxmt) [inline foreign-key.md:301]: @frank945946 PTAL

**Review by hfxsd — APPROVED** [review]

**Review by Frank945946 — APPROVED** [review]

**Review by shichun-0415 — APPROVED** [review]
LGTM

**Pull request merged**